src/client/js/script.js

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Because Merge was not made right.
This indicates where was your head when the merge conflict happened.